### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by HLWeil** [nfdi4plants/isa-ro-crate-profile#3]
Currently, the whole specification of how to handle `Comments` is a bit messy. In ISA, a lot of types have a property called `Comments` of type [ISA Comment](https://isa-specs.readthedocs.io/en/latest/isajson.html#comment-schema-json) array. 
Unfortunately, not all of their equivalent types in Schema.org contain a property `comment` of range https://schema.org/Comment. So for some cases like https://schema.org/Person, we use `disambiguatingDescription` of range `Text`. But this is not ideal, as now the encoders for the same type in ISA needs to produce different objects in JsonLD. 
Additionally, currently the [Investigation profile](https://www.researchobject.org/ro-crate/profiles.html#isa-investigation-profile) defines  `disambiguatingDescription`, even though this is inconsistent with the other `CreativeWork` objects.

Not sure how to handle this. Just wanted to write it down before it gets forgotten.

**Comment by floWetzels** [comment]
> Currently, the whole specification of how to handle `Comments` is a bit messy. In ISA, a lot of types have a property called `Comments` of type [ISA Comment](https://isa-specs.readthedocs.io/en/latest/isajson.html#comment-schema-json) array. Unfortunately, not all of their equivalent types in Schema.org contain a property `comment` of range https://schema.org/Comment. So for some cases like https://schema.org/Person, we use `disambiguatingDescription` of range `Text`.

I agree, it is not nice at the moment. But I think there is no better way around this Problem. 


> But this is not ideal, as now the encoders for the same type in ISA needs to produce different objects in JsonLD.

This is just a technical problem, right?  I don't think this is the right place to discuss it, but from my point of view, it's not a big hurdle. We just define two different encoders for the ISA type `Comment`, so there should be no problem with them producing different json objects.


> Additionally, currently the [Investigation profile](https://www.researchobject.org/ro-crate/profiles.html#isa-investigation-profile) defines `disambiguatingDescription`, even though this is inconsistent with the other `CreativeWork` objects.

That is indeed a bug, as far as I can tell. Same holds for `ScholarlyArticle`. @stuzart, do you know if this is by design or just a bug?

**Comment by floWetzels** [comment]
The actual bug regarding Investigation and ScholarlyArticle has been fixed.
@HLWeil and I agreed on specificying the usage of `disambiguatingDescription` more precisely in the profile where it needs to be used for comments. I will take care of this.

**Comment by HLWeil** [comment]
Fixed over various PRs
